packages/rollup-full-node/test/app/web-rpc-handler.spec.ts

```diff
@@ -9,7 +9,7 @@ import {
   ZERO_ADDRESS,
   hexStrToBuf,
 } from '@eth-optimism/core-utils'
-import { CHAIN_ID } from '@eth-optimism/ovm'
+import { CHAIN_ID, convertInternalLogsToOvmLogs } from '@eth-optimism/ovm'
```

Comment:
I think this can be reverted. If it was used, it'd show up in a diff, right?

Reply:
Oh yeah that's definitely unused